### ISSUE TITLE
new adjustment for Egyptian fajr angles

### ISSUE THREAD

**Issue by Abu-Abdullah** [batoulapps/adhan-kotlin#14]
salam Alikom,

few new adjustments in:
https://github.com/batoulapps/adhan-java/blob/master/adhan/src/main/java/com/batoulapps/adhan/CalculationMethod.java

Egyptian General Authority of Survey
new CalculationParameters(19.5, 17.5, this)

currently:
new CalculationParameters(20.0, 18.0, this)

it seems also swift version is more extensive: 
https://github.com/batoulapps/Adhan/blob/4c2b3828b31eb16e2bd54b8f1a0cff0d892a5150/Swift/Adhan/Adhan.swift

the numbers also seems not matching there as well e.g. UMM_AL_QURA 18.0 where it is in java 18.5

**Comment by ahmedre** [comment]
wa3laikum alsalam,
for Egypt looks like there are now two:

```swift
	 case .egyptianGeneralAuthority:
          return CalculationParameters(fajrAngle: 19.5, ishaAngle: 17.5, method: self)
          
        case .egyptianGeneralNewAuthority:
          return CalculationParameters(fajrAngle: 20, ishaAngle: 18, method: self)
```

i am seeing it as 18.5 for Swift version also:

```swift
        case .ummAlQura:
          return CalculationParameters(fajrAngle: 18.5, ishaInterval: 90, method: self)
          
        case .ummAlQuraRamadan:
          return CalculationParameters(fajrAngle: 18.5, ishaInterval: 120, method: self)
```

will try to make sure the java library is in sync with the Swift one in sha' Allah.

**Comment by ahmedre** [comment]
actually just double checked - these extra methods aren't in the latest Swift version - cc @z3bi am i missing anything or were these removed some time back?

**Comment by z3bi** [comment]
Salam, the file @Abu-Abdullah is linking to is from this PR https://github.com/batoulapps/Adhan/pull/59 that was never actually merged in.

Please look at the head branch of the Swift version for the correct values. It seems the only one that is wrong is the Egyptian method. I think that value may have been adjusted at one version and forgot to get updated in this repo.

The missing methods are the Tehran method and the Turkey method.

The Turkey method is straightforward to add, it is important to mention in the README that this version of the Diyanet method only works for Turkey (unfortunately I'm not sure how diyanet gets adjusted for other locations).

The Tehran method requires adding a new parameter of maghribAngle, but that shouldn't be too difficult.

**Comment by ahmedre** [comment]
wa3laikum alsalam,
fixed the Egyptian method in #29
